example/README.md

```diff
@@ -104,3 +104,6 @@ You can view the [transaction details in Square Dashboard].
 [Flutter Getting Started]: https://flutter.io/docs/get-started/install
 [root README]: ../README.md
 [transaction details in Square Dashboard]: https://squareup.com/dashboard/sales/transactions
+
+## Feedback
+Leave a feedback for this sample app [here](https://delighted.com/t/DT6msOcY)!
```

Comment:
nit: I think it sounds better to say `Leave feedback for this sample app`

Reply:
The survey language changed, so I will change it to "Rate this sample app here”. Lmk if you'd prefer an alternative